source/docs/advanced/server/configurations.md

```diff
@@ -0,0 +1,246 @@
+---
+id: setup-configurations
+title: Setup configurations
```

Comment:
I am not sure that people would understand from the title what this section is about (maybe because it is not my mother-tongue). I don't have any final suggestion for it though.

Maybe "Configurations" would be sufficient.

Or
```suggestion
title: Configure server modules
```